packages/app/src/components/MenuBar/styles/MenuBar.style.js

```diff
@@ -11,11 +11,6 @@ export default theme => ({
     updateButton: {
         marginRight: theme.spacing.unit,
     },
-    label: {
```

Comment:
Convert this to a module CSS file?

Reply:
Could do, just didnt think it was in scope 🤷‍♂️  but its an easy fix 🙂

Reply:
Fixed